extensions/blocks/opentable/edit.js

```diff
@@ -0,0 +1,294 @@
+						onClick: () => updateStyle( styleOption.value ),
+					} ) ) }
+				/>
+			) }
+		</BlockControls>
+	);
+
+	const onPickerSubmit = input => {
+		if ( Array.isArray( input ) ) {
+			setAttributes( {
+				rid: input,
+				style: input.length > 1 && 'button' === style ? 'standard' : style,
+			} );
+		} else {
+			parseEmbedCode( input );
+		}
+	};
+
+	const inspectorControls = () => (
+		<InspectorControls>
+			<PanelBody title={ __( 'Styles', 'jetpack' ) }>
+				<div className="block-editor-block-styles">
+					{ styleOptions.map( styleOption => {
+						return (
+							<div
+								key={ styleOption.value }
+								className={ classnames( 'block-editor-block-styles__item is-opentable', {
+									'is-active': styleOption.value === style,
+								} ) }
+								onClick={ () => updateStyle( styleOption.value ) }
+								onKeyDown={ event => {
+									if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
+										event.preventDefault();
+										updateStyle( styleOption.value );
+									}
+								} }
+								role="button"
+								tabIndex="0"
+								aria-label={ styleOption.label }
+							>
+								<div className="block-editor-block-styles__item-preview is-opentable">
+									{ blockPreview( styleOption.value ) }
+								</div>
+								<div className="block-editor-block-styles__item-label">{ styleOption.label }</div>
+							</div>
+						);
+					} ) }
+				</div>
+			</PanelBody>
+			<PanelBody title={ __( 'Settings', 'jetpack' ) }>
+				<RestaurantPicker rids={ rid } onChange={ onPickerSubmit } />
+				<SelectControl
+					label={ __( 'Language', 'jetpack' ) }
+					value={ lang }
+					onChange={ newLang => setAttributes( { lang: newLang } ) }
+					options={ languageOptions }
+				/>
+				<ToggleControl
+					label={ __( 'Load the widget in an iFrame (Recommended)', 'jetpack' ) }
+					checked={ iframe }
+					onChange={ () => setAttributes( { iframe: ! iframe } ) }
+				/>
+				<ToggleControl
+					label={ __( 'Open in a new window', 'jetpack' ) }
+					checked={ newtab }
+					onChange={ () => setAttributes( { newtab: ! newtab } ) }
+				/>
+			</PanelBody>
+			<PanelBody title={ __( 'Embed code', 'jetpack' ) } initialOpen={ false }>
+				<EmbedCodeForm onSubmit={ parseEmbedCode } />
+			</PanelBody>
+		</InspectorControls>
+	);
+
+	const blockPlaceholder = (
+		<Placeholder
+			label={ __( 'OpenTable Reservation', 'jetpack' ) }
+			icon={ <BlockIcon icon={ icon } /> }
+			notices={
+				notice && (
+					<Notice status="error" isDismissible={ false }>
+						{ notice }
+					</Notice>
+				)
+			}
+		>
+			<RestaurantPicker
+				rids={ rid }
+				label={ __(
+					'Enter your restaurant name, OpenTable Restaurant ID or embed code',
+					'jetpack'
+				) }
+				onSubmit={ onPickerSubmit }
+			/>
+			<div className={ `${ className }-placeholder-links` }>
+				<ExternalLink
+					href="https://en.support.wordpress.com/widgets/open-table-widget/"
+					target="_blank"
```

Comment:
I don't know if you need the `target` attribute here, since the component is already `ExternalLink`? I would assume it already handles that for you, but I have not tested.

Reply:
You're right, it's already on the link